### PR TITLE
JDF-781: Update EAP product name to use Red Hat prefix

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: kitchensink-html5-mobile/.cheatsheet.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
...
+      As usual, let&apos;s start by looking at the necessary deployment descriptors. By now, we&apos;re very used to seeing <b>beans.xml</b> in <b>WEB-INF/</b> (which can be found in the <b>src/main/webapp</b> directory of the example). Notice that, once again, we don&apos;t need a web.xml. There are two configuration files (which can be found in the <b>src/main/resources</b> directory of the example) — <b>WEB-INF/classes/META-INF/persistence.xml</b>, which sets up JPA, and <b>WEB-INF/classes/import.sql</b> which Hibernate, the JPA provider in Red Hat JBoss Enterprise Application Platform, will use to load the initial users into the application when the application starts. We discussed both of these files in detail in The <b>greeter example in depth</b>, and these are largely the same.
```
Comment (LightGuard) [inline kitchensink-html5-mobile/.cheatsheet.xml:56]: What changed here, not seeing it.

**Comment by LightGuard** [comment]
Aside from my comment above, looks good

File: kitchensink-html5-mobile/.cheatsheet.xml (hunk 1 above)
Comment (sgilda) [inline kitchensink-html5-mobile/.cheatsheet.xml:56]: I added the 'Red Hat' prefix to 'JBoss Enterprise Application Platform'

**Comment by sgilda** [comment]
Thanks @LightGuard !

**Pull request merged**